spec.html

```diff
@@ -26620,6 +26620,28 @@ <h1>Abstract Module Records</h1>
             </tr>
           </table>
         </emu-table>
+
+        <emu-clause id="sec-EvaluateModuleSync" type="abstract operation">
+          <h1>
+            EvaluateModuleSync (
+              _module_: a Module Record,
+            ): either a normal completion containing ~unused~ or a throw completion
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It synchronously evaluates _module_, provided that the caller guarantees that _module_'s evaluation will return an already settled promise.</dd>
+          </dl>
+
+          <emu-alg>
+            1. Assert: _module_ is not a Cyclic Module Record.
+            1. Let _promise_ be ! _module_.Evaluate().
```

Comment:
Pre-existing bug: `.Evaluate` can only return a Promise, not a completion record, so the `!` doesn't apply.
```suggestion
            1. Let _promise_ be _module_.Evaluate().
```